### PR TITLE
add new theme Abyssal Wave

### PR DESCRIPTION
**Pull request by Itz-Abhishek-Tiwari**
Added Abyssal Wave Theme and fix the generate readme  file instruction

### CODE REVIEW

File: json
Comment (kRHYME7) [inline json:1]: what's this?

File: generate_readme.py
Comment (kRHYME7) [inline generate_readme.py:1]: also this one?

**Review by kRHYME7 — CHANGES_REQUESTED** [review]

File: json
Comment (Itz-Abhishek-Tiwari) [reply json:1]: well i don't know it generate when i run the `python generate_readme.py` as `./generate_readme.py` is not runing

**Review by kRHYME7 — APPROVED** [review]
Thanks

**Pull request merged**